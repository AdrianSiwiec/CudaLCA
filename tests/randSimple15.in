15
0
1
1
1
3
0
5
5
2
4
7
6
8
9
15
4 7
3 12
12 13
13 1
7 8
3 11
2 3
11 4
12 12
6 4
9 12
9 0
9 8
11 10
9 0
