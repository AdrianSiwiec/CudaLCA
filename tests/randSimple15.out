1
0
0
1
5
3
1
1
12
0
0
0
1
1
0
